test/blackbox-tests/test-cases/melange/merlin.t

```diff
@@ -35,5 +42,18 @@
 
   $ touch main.ml
   $ dune build @check
-  $ dune ocaml merlin dump-config "$PWD" | grep -i "$target"
+  $ dune ocaml merlin dump-config $PWD | grep -i "$target"
     $TESTCASE_ROOT/_build/default/.output.mobjs/melange)
+
+The melange.emit entry contains a ppx directive
+
+  $ dune ocaml merlin dump-config $PWD | grep -i "ppx"
+   (FLG (-ppx "melc -as-ppx -bs-jsx 3"))
+
+  $ dune ocaml dump-dot-merlin $PWD
+  EXCLUDE_QUERY_DIR
+  STDLIB /home/me/code/dune/_opam/lib/melange
+  B $TESTCASE_ROOT/_build/default/.output.mobjs/melange
+  S $TESTCASE_ROOT
+  # FLG -w @1..3@5..28@30..39@43@46..47@49..57@61..62@67@69-40 -strict-sequence -strict-formats -short-paths -keep-locs
+  # FLG -ppx melc -as-ppx -bs-jsx 3
```

Comment:
I think this needs to be `FLG -ppx 'melc -as-ppx -bs-jsx 3'`

Reply:
Fixed in https://github.com/ocaml/dune/pull/6476/commits/5c34973f817c0dab8da113fcb6849b3898eb0520.